trollflow2/launcher.py

```diff
@@ -217,6 +217,29 @@ def expand(yml):
     return yml
 
 
+def get_dask_client(config):
+    """Create Dask client if configured."""
+    client = None
+
+    try:
+        client_class = config["dask_distributed"]["class"]
+        client = client_class(**config["dask_distributed"]["settings"])
+        try:
+            if not client.ncores():
+                LOG.error("No workers available, reverting to default scheduler")
```

Comment:
What about just warning ? Maybe the workers are started after ?

Reply:
They could be started later, but we aren't waiting for them. But I'll make it a warning in any case.